sensor_properties[] valid_messages
